pkg/networkmanager/team.jsx

```diff
@@ -170,7 +170,7 @@ export const TeamDialog = ({ connection, dev, settings }) => {
                         {team_balancer_choices.map(choice => <FormSelectOption value={choice.choice} label={choice.title} key={choice.choice} />)}
                     </FormSelect>
                 </FormGroup>}
-                {runner == "active-backup" && <FormGroup fieldId={idPrefix + "-primary-select"} label={_("Primary")}>
+                {runner == "activebackup" && <FormGroup fieldId={idPrefix + "-primary-select"} label={_("Primary")}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19003-20230623-085735-84dc87dd-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)